spec/index.bs

```diff
@@ -3898,6 +3930,38 @@ dictionary GPURenderPassDescriptor : GPUObjectDescriptorBase {
 };
 </script>
 
+<dl dfn-type=dict-member dfn-for=GPURenderPassDescriptor>
+    : <dfn>colorAttachments</dfn>
+    ::
+        The set of {{GPURenderPassColorAttachmentDescriptor}} values in this sequence defines which
+        color attachments will be output to when executing this render pass.
+
+    : <dfn>depthStencilAttachment</dfn>
+    ::
+        The {{GPURenderPassDepthStencilAttachmentDescriptor}} value that defines the depth/stencil
+        attachment that will be output to and tested against when executing this render pass.
+
+    : <dfn>occlusionQuerySet</dfn>
+    ::
+        Issue: Describe this dictionary member
+</dl>
+
+<div class=validusage dfn-for=GPURenderPassDescriptor>
+    <dfn abstract-op>Valid Usage</dfn>
+
+    Given a {{GPURenderPassDescriptor}} |this| the following validation rules apply:
+
+    1. |this|.{{GPURenderPassDescriptor/colorAttachments}}.length must greater than `0` or
+        |this|.{{GPURenderPassDescriptor/depthStencilAttachment}} must not be `null`.
+    1. For each |colorAttachment| in |this|.{{GPURenderPassDescriptor/colorAttachments}}:
```

Comment:
From memory in Dawn we found we needed these constraints:

 - All attachments (color and depth-stencil) must also have the same sample count and size.
 - All attachments must be a single subresource.

Reply:
>All attachments must be a single subresource.

I'm not entirely sure what this means?

Reply:
It's [defined in the spec](https://gpuweb.github.io/gpuweb/#subresource). A texture view sees a subset of texture's subresources. In this case, the set should be only a single subresource.

Reply:
I think you missed the part where the attachments all have the same size.